### DISCUSSION TITLE
Custom Icons for Essential tabs

### DISCUSSION THREAD

**Discussion by cdfasnacht** [zen-browser/desktop#3211] [Ideas]
As a user of Zen Browser and a web developer
I created essential tabs for the various environments of an application that I'm working on

I would like to be able to customize the icons of those tabs so that I can easily tell which environment is which. I can't change the favicons for each of the environments so a feature in Zen to customize the icon for essential tabs would be great to have.

**Comment by niccy266** [comment]
I would also like this feature! Maybe also an option to make icons persistent? save to hard drive somewhere? sometimes when i open zen the icons are missing and i have to load the page for the icons to restore
